frontend/src/libs/helpers/is-route-selected.ts

```diff
@@ -0,0 +1,16 @@
+import { AppRoute } from '#libs/enums/app-route.enum.js';
+
+type Properties = {
+  pathname: string;
+  routePath: string;
+};
+
+const isRouteSelected = ({ pathname, routePath }: Properties): boolean => {
+  const isRootPathSelected = pathname === routePath;
+  const isOtherPathSelected =
```

Comment:
maybe let's name it like this?
```suggestion
  const isChildPathSelected =
```

Reply:
Why would it be a child?🤔
As in non root?

Reply:
as I understood, root path can be /chats and other path is /chats/2 for example, right?

Reply:
if so, it is not so clear, what is the other path name means. The other path can be /journal for example. So that's why I suggested to name it like isRootChildPathSelected or similar

Reply:
Yes, that makes sense!